doc/source/index.rst

```diff
@@ -2,19 +2,33 @@
 Satpy's Documentation
 =====================
 
-Satpy is a python library for reading and manipulating
-meteorological remote sensing data and writing it to various image and
-data file formats. Satpy comes with the ability to make various RGB
-composites directly from satellite instrument channel data or higher level
-processing output. The
-`pyresample <http://pyresample.readthedocs.io/en/latest/>`_ package is used
-to resample data to different uniform areas or grids. Various atmospheric
-corrections and visual enhancements are also provided, either directly in
-Satpy or from those in the
-`PySpectral <https://pyspectral.readthedocs.io/en/develop/>`_ and
-`TrollImage <http://trollimage.readthedocs.io/en/latest/>`_ packages.
-
-Go to the project_ page for source code and downloads.
+Satpy is a python library for reading, manipulating, and writing data from
+remote-sensing earth-observing satellite instruments. Satpy provides users
+with a series of readers that convert satellite instrument file formats to
```

Comment:
What about a little physics here: "Satpy provides users with a series of readers that convert raw satellite data to physical quantities such as radiance, reflectance or brightness temperature (so called calibration). The calibrated data is returned in the common Xarray..."

Reply:
We have readers for other types of data (cloud masks, fires, other types of temperatures, cloud heights, etc). Does telling users about radiance/reflectance/BTs add anything to the description? I'm worried that it hurts the "executive summary" nature of the paragraph...but then again I mention Xarray objects.

Reply:
Good point, I was too narrow-minded there. What about "Satpy provides users with a series of readers that convert a multitude of geophysical parameters in different file formats to the common Xarray DataArray and Dataset classes". I think mentioning Xarray is a good thing.

Reply:
Maybe "satellite-derived geophysical parameters"

Reply:
Or satellite-borne?

Reply:
I like @sfinkens 's proposal